Formula/libgr.rb

```diff
@@ -0,0 +1,48 @@
+class Libgr < Formula
+  desc "GR framework: a graphics library for visualisation applications"
+  homepage "https://gr-framework.org/"
+  url "https://github.com/sciapp/gr/archive/v0.37.0.tar.gz"
+  sha256 "9120c4215cd2c10239f45f4cbf79dcb199a9e7e3bbbf36e5ce8f4e5eb2e70e24"
+  depends_on :xcode => :build
+  depends_on "cairo"
+  depends_on :x11
```

Comment:
If it can work with Quartz or OpenGL windows, or with anything other than X11, let's avoid adding this dependency please.

Reply:
I do think it works with quartz, so the correct thing is to just remove this declaration or change it to something like `depends_on :quartz`?

Reply:
Remove X11.